examples/chem-sync-local-flask/README.md

```diff
@@ -95,17 +98,17 @@ When prompted to upload a file, select `manifest.yaml` and click "Create."
 
 ### Update the Webhook URL
 
-Every time we restart the `local-tunnel` Docker container, it will provision
+Every time we restart the `cloudflare-tunnel` Docker container, it will provision
 a new public webhook URL.
 
 Update the Benchling App's Webhook URL in the UI with the new server and
 append the path our Flask route expects (see `local_app/app.py`).
 
-For example, if our `localtunnel` generated URL is `https://hot-ideas-doubt.loca.lt`,
+For example, if our `cloudflare` generated URL is `https://hot-ideas-doubt.trycloudflare.com`,
 the webhook URL in Benchling should be:
 
 ```
-https://hot-ideas-doubt.loca.lt/1/webhooks
+https://hot-ideas-doubt.trycloudflare.com/1/webhooks
 ```
 
 ![image info](./docs/update-webhook-url.gif)
```

Comment:
there's a line a bit later in this readme
```
If you restart both containers, be sure to update your App in Benchling with the new webhook URL from localtunnel.
```
can we just change the last word to `cloudflare-tunnel`?

Reply:
Yes, good call.